inttests/storagepool_test.go

```diff
@@ -223,3 +223,62 @@ func TestCreateDeleteStoragePool(t *testing.T) {
 	assert.NotNil(t, err)
 
 }
+
+// TestGetSDSStoragePool gets the SDS instances associated with storage pool
+func TestGetSDSStoragePool(t *testing.T) {
+	pool := getStoragePool(t)
+	assert.NotNil(t, pool)
+
+	stats, err := pool.GetSDSStoragePool()
+	assert.Nil(t, err)
+	assert.NotNil(t, stats)
+}
+
+// Modify TestModifyStoragePoolName
+func TestModifyStoragePoolName(t *testing.T) {
+	domain := getProtectionDomain(t)
+	assert.NotNil(t, domain)
+	_, err := domain.ModifyStoragePoolName("Invalid", "STPnew")
+	assert.NotNil(t, err)
+}
+
+// Modify TestStoragePoolMediaType
+func TestStoragePoolMediaType(t *testing.T) {
+	domain := getProtectionDomain(t)
+	assert.NotNil(t, domain)
+	_, err := domain.ModifyStoragePoolMedia("b9b0be6600000004", "SSD")
+	assert.Nil(t, err)
+}
+
+// Modify TestEnableRFCache
+func TestEnableRFCache(t *testing.T) {
+	domain := getProtectionDomain(t)
+	assert.NotNil(t, domain)
+	_, err := domain.EnableRFCache("b9b0be6400000003")
+	assert.Nil(t, err)
+}
+
+// Modify TestDisableRFCache
+func TestDisableRFCache(t *testing.T) {
+	domain := getProtectionDomain(t)
+	assert.NotNil(t, domain)
+	_, err := domain.DisableRFCache("b9b0be6400000003")
+	assert.Nil(t, err)
+}
+
+// Set TestSetRmcache
+func TestSetRmcache(t *testing.T) {
```

Comment:
If i understood correct this function tests the modifyRmcache fn right? Please Rename it to TestModifyRmCache

Reply:
Done